multitool/defs.bzl

```diff
@@ -0,0 +1,54 @@
+"""fork of rules_multitool
+Derived from https://github.com/theoremlp/rules_multitool
+but simplified to just generate an alias without any toolchains.
+"""
+
+_COMMON_FILES_TO_GENERATE = [
+    "BUILD.bazel",
+    "WORKSPACE.bazel",
+]
+
+_PLATFORMS = [
```

Comment:
This implies all tools have these 4 platforms and only these? ~~If~~ When we have a tool that has more/less will it require a change to a public API? (if `multitool/defs.bzl` is non-public then this doesn't matter atm)

Reply:
bazel-lib already has a bunch of "download a tool and make it a toolchain" so maybe there's a good reason to make a side effort here to consolidate the options and have a way that scales to "lots" of tools.

Reply:
trying again with rules_multitool. It still has only these four platforms, but I think we can add more without a breaking change. Just means a platform like windows was previously an error, and then can start working.

Reply:
Happy to take a PR to extend os/architecture set -- recent changes should make the architectures pretty straight forward. We don't really have a setup to test on Windows, so might be a bigger lift to add.

Reply:
We can contribute for Windows, thanks!